src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx

```diff
@@ -105,38 +109,56 @@ class OfferTrade extends Component {
-            <p className="mt-3 limits">Limits: {limitDecimals(minFiat)} {currency.id} to <span id="max-token">{limitDecimals(maxFiat)} {currency.id}</span></p>
+            { amountGreaterThanBalance && <FormFeedback className="d-block">{t('buyer.offerTrade.amountWarning')}</FormFeedback> }
+            <p className="mt-3 limits">
+              <Trans 
+               i18nKey="buyer.offerTrade.limits" values={{
+                min: limitDecimals(minFiat),
+                max: limitDecimals(maxFiat),
+                symbol: currency.id
+              }}>
+              Limits: 0 USD to <span id="max-token">N USD</span>
+              </Trans>
+            </p>  
           </Fragment>
         }
         {disabled && <p className="text-muted">{t('buyer.offerTrade.enterBefore')}</p>}
         {notEnoughETH && !canRelay && <Col xs="12" className="text-small text-center text-danger">
-              New orders can be created in {moment(escrow.helpers.nextRelayDate(lastActivity)).toNow(true)}
+          {t('buyer.offerTrade.newOrderDelay', {time: moment(escrow.helpers.nextRelayDate(lastActivity)).toNow(true)})}
         </Col>}
       </Form>
     </Col>
   </Row>
-  <ModalDialog display={this.state.displayDialogArbitrator} onClose={this.toggleArbitratorDialog} buttonText="Ok got it">
+  <ModalDialog display={this.state.displayDialogArbitrator} onClose={this.toggleArbitratorDialog} buttonText={t('buyer.offerTrade.dialogButton')}>
     <RoundedIcon image={arbitratorImg} className="mb-2" bgColor="blue" />
-    <h2>Arbitrator</h2>
-    <p className="text-muted">You can think of an arbitrator as a neutral <span className="text-black">judge</span> who, when the trade comes to the <span className="clickable" onClick={this.toggleDisputeDialog}>dispute</span>, will review the trade and resolve the dispute.</p>
+    <h2>{t('general.arbitrator')}</h2>
+    <Trans i18nKey="buyer.offerTrade.arbitratorInfo">
+      <p className="text-muted">You can think of an arbitrator as a neutral <span className="text-black">judge</span> who, when the trade comes to the <span className="clickable text-black" onClick={this.toggleDisputeDialog}>dispute</span>, will review the trade and resolve the dispute.</p>
+    </Trans>
   </ModalDialog>
-  <ModalDialog display={this.state.displayDisputeDialog} onClose={this.toggleDisputeDialog} buttonText="Ok got it">
+  <ModalDialog display={this.state.displayDisputeDialog} onClose={this.toggleDisputeDialog} buttonText={t('buyer.offerTrade.dialogButton')}>
     <RoundedIcon image={disputeImg} className="mb-2" bgColor="blue" />
-    <h2>Dispute</h2>
-    <p className="text-muted">A dispute is a disagreement on a point of law or fact, a conflict of legal views or of interests between two persons.</p>
+    <h2>{t('buyer.offerTrade.dispute')}</h2>
+    <p className="text-muted">{t('buyer.offerTrade.disputeInfo')}</p>
```

Comment:
Did I completely miss this? I guess I just skipped it by accident

Reply:
no big deal. We'll capture anything missing when we create a new language file!